common/project_manager.py

```
@@ -50,13 +47,13 @@ class ChallengeData(BaseModel):
 class ChallengeResponse(BaseModel):
     nextUpdate: int
     now: int
-    boardChallenges: List[ChallengeData]
+    boardChallenges: list[ChallengeData]
 
 ALLOWED_CID = []
 
 class ProjectManager:
-    projects: Dict[str, Project] = {}
-    projects_config: Dict[str, ProjectConfig] = {}
+    projects: dict[str, Project] = {}
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)